app/views/layouts/_navigation.html.erb

```diff
@@ -1,6 +1,20 @@
-<nav class="nav nav-style nav-txt-color p-2 nav-items-container col-md-12">
-  <%= link_to 'Development Metrics', development_metrics_path, class: 'nav-link active nav-txt-color'%>
-  <%= link_to 'Tech Blog', tech_blog_path, class: 'nav-link active nav-txt-color'%>
-  <%= link_to 'Open Source', open_source_path, class: 'nav-link active nav-txt-color'%>
-  <%= link_to 'Admin', admin_events_path, class: 'nav-link active nav-txt-color admin-button'%>
+<nav class="nav nav-style nav-txt-color p-2 nav-items-container col-md-12 d-flex justify-content-between">
+  <div class="d-flex flex-row">
+    <%if current_page?(development_metrics_path)%>
+      <%= link_to 'Development Metrics', development_metrics_path, class: 'nav-link active nav-txt-color selected mx-1'%>
+    <%else%>
+      <%= link_to 'Development Metrics', development_metrics_path, class: 'nav-link active nav-txt-color mx-1'%>
+    <%end%>
+    <%if current_page?(tech_blog_path)%>
+      <%= link_to 'Tech Blog', tech_blog_path, class: 'nav-link active nav-txt-color selected mx-1'%>
+    <%else%>
+      <%= link_to 'Tech Blog', tech_blog_path, class: 'nav-link active nav-txt-color mx-1'%>
+    <%end%>
+    <%if current_page?(open_source_path)%>
+      <%= link_to 'Open Source', open_source_path, class: 'nav-link active nav-txt-color selected mx-1'%>
+    <%else%>
+      <%= link_to 'Open Source', open_source_path, class: 'nav-link active nav-txt-color mx-1'%>
+    <%end%>
+  </div>
+  <%= link_to 'Admin', admin_events_path, class: 'nav-link active nav-txt-color mx-1'%>
```

Comment:
im not sure if using a ternary operator would be better

Reply:
There's probably a DRYer way of doing it but I'm ok with this